beacon-chain/db/kv/validated_checkpoint.go

```diff
@@ -0,0 +1,47 @@
+package kv
+
+import (
+	"context"
+	"encoding/binary"
+
+	"github.com/pkg/errors"
+	types "github.com/prysmaticlabs/eth2-types"
+	"github.com/prysmaticlabs/prysm/encoding/bytesutil"
+	bolt "go.etcd.io/bbolt"
+	"go.opencensus.io/trace"
+)
+
+func (s *Store) LastValidatedCheckpoint(ctx context.Context) ([32]byte, types.Slot, error) {
```

Comment:
This is not exported in the declaration

Reply:
I assume that this function will be used outside DB pkg for getting the latest validated checkpoint and its slot.

PS: i forgot to export the save function... i will export that too